ivy/functional/backends/tensorflow/statistical.py

```diff
@@ -8,6 +8,7 @@
 from ivy.functional.ivy.statistical import _get_promoted_type_of_operands
 from ivy.func_wrapper import with_unsupported_dtypes
 from . import backend_version
+from ivy.utils.exceptions import IvyNotImplementedException
```

Comment:
This should not be imported as we should be implementing `cummax` in TensorFlow backend.